README.md

```diff
@@ -201,3 +201,13 @@ $ mvn clean install
 ```
 
 Also see our [Contributing Policy](CONTRIBUTING.md) and [Code of Conduct](CODE_OF_CONDUCT.md)
+
+## White paper
```

Comment:
I think we can have a separate readme inside `wp` directory with these build instructions, not to overload the main readme.